doc-deploy-dev/action.yml

```diff
@@ -15,6 +15,39 @@ inputs:
     description: "Required token for documentation deployment."
     required: true
     type: string
+  workflow:
+    description: "Workflow name from where the artifacts are downloaded. By default, the same running workflow"
+    required: false
+    default: ""
+    type: string
+  repo:
+    description: Repository name with owner (like actions/checkout)
+    required: false
+    default: ${{ github.repository }}
+  pr:
+    description: Pull request number
+    required: false
+  commit:
+    description: Commit hash
+    required: false
+  branch:
+    description: Branch name
+    required: false
+  event:
+    description: Event type
+    required: false
+  run_id:
+    description: Workflow run id
+    required: false
+  run_number:
+    description: Workflow run number
+    required: false
+  name:
+    description: Artifact name (download all artifacts if not specified)
+    required: false
+  dry_run:
+    description: Check the existence of artifact(s) without downloading
+    required: false
```

Comment:
Please, make sure that each input has the following fields:

* Description: "...." (Use single quotes
* Default (if required): default-value
* Required: true/false
* Type: [boolean, string...]

Reply:
This is now resolved.